### PR TITLE
feat: CB-3517 css-modules for styling components

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: webapp/packages/core-blocks/src/Menu/MenuItem.tsx [hunk 1]
```diff
@@ -8,46 +8,44 @@
...
   selected?: boolean;
   close?: boolean;
-  style?: ComponentStyle;
-}
+};
 
-export const MenuItem = observer<IMenuItemProps>(function MenuItem({ label, children, hidden, selected, close, style, onClick, ...rest }) {
-  const menu = useContext(MenuStateContext);
-  const styles = useStyles(menuPanelStyles, style);
+export const MenuItem: ReakitProxyComponent<'button', MenuItemOptions> = observer<ReakitProxyComponentOptions<'button', MenuItemOptions>>(
```
Comment (devnaumov) [inline webapp/packages/core-blocks/src/Menu/MenuItem.tsx:25]: Do we need to duplicate type observer<ReakitProxyComponentOptions<'button', MenuItemOptions>>?

**Review by devnaumov — APPROVED** [review]

File: webapp/packages/core-blocks/src/FormControls/Checkboxes/CheckboxMarkup.m.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.checkbox {
```
Comment (tfokina) [inline webapp/packages/core-blocks/src/FormControls/Checkboxes/CheckboxMarkup.m.css:1]: should it be empty?

File: webapp/packages/core-ui/src/ContextMenu/MenuBar/MenuBar.m.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.menuBar {
+  /*composes: theme-background-secondary theme-text-on-secondary theme-border-color-background theme-typography--body2 from global;*/
```
Comment (tfokina) [inline webapp/packages/core-ui/src/ContextMenu/MenuBar/MenuBar.m.css]: unnecessary comment?

File: webapp/packages/plugin-datasource-context-switch/src/ConnectionSchemaManager/ConnectionSelector/ConnectionIcon.tsx [hunk 4]
```diff
@@ -55,15 +65,17 @@ export const ConnectionIcon: React.FC<Props> = observer(function ConnectionIcon(
...
-    <icon className={className}>
-      <ConnectionImageWithMask
-        icon={driver.icon}
-        connected={connection.data.connected}
-        maskId="connection-icon"
-        size={24}
-        paddingSize={4}
-        {...use({ small })}
-      />
+    <icon className={clsx(ConnectionImageWithMaskSvgBackgroundStyles.connectionIcon, className)}>
```
Comment (tfokina) [inline webapp/packages/plugin-datasource-context-switch/src/ConnectionSchemaManager/ConnectionSelector/ConnectionIcon.tsx]: maybe use `s(...)` function here?

File: webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/TopMenu.m.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
+/*Button:disabled,
```
Comment (tfokina) [inline webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/TopMenu.m.css]: unnecessary comment?

**Review by tfokina — APPROVED** [review]

File: webapp/packages/core-blocks/src/Menu/MenuItem.tsx (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core-blocks/src/Menu/MenuItem.tsx:25]: yep

File: webapp/packages/core-blocks/src/FormControls/Checkboxes/CheckboxMarkup.m.css (hunk 2 above)
Comment (Wroud) [reply webapp/packages/core-blocks/src/FormControls/Checkboxes/CheckboxMarkup.m.css:1]: yep

File: webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/TopMenu.m.css (hunk 5 above)
Comment (Wroud) [reply webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/TopMenu.m.css]: removed

File: webapp/packages/plugin-datasource-context-switch/src/ConnectionSchemaManager/ConnectionSelector/ConnectionIcon.tsx (hunk 4 above)
Comment (Wroud) [reply webapp/packages/plugin-datasource-context-switch/src/ConnectionSchemaManager/ConnectionSelector/ConnectionIcon.tsx]: replaced

File: webapp/packages/core-ui/src/ContextMenu/MenuBar/MenuBar.m.css (hunk 3 above)
Comment (Wroud) [reply webapp/packages/core-ui/src/ContextMenu/MenuBar/MenuBar.m.css]: removed

**Pull request merged**